src/Controller/ApiV1Controller.php

```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * Tatoeba Project, free collaborative creation of multilingual corpuses project.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * @category PHP
+ * @package  Tatoeba
+ * @license  Affero General Public License
+ * @link     https://tatoeba.org
+ */
+namespace App\Controller;
+
+use App\Controller\AppController;
+use App\Form\SentencesSearchForm;
+use App\Model\CurrentUser;
+use Cake\Core\Configure;
+use Exception;
+
+/**
+ * Controller for Api.
+ *
+ * @category Sentences
+ * @package  Controllers
+ * @author   githubshrek
+ * @license  Affero General Public License
+ * @link     https://tatoeba.org
+ */
+class ApiV1Controller extends AppController
```

Comment:
Any objection to renaming this into `ApiBetaController` or `ApiV0Controller`? I'm just thinking that this is still at a quite experimental stage, and I'd rather keep `v1` for when we have something more stable :)

Reply:
I'm OK to make this change. To be honest, I think we're just as likely to end up with v0 for the next 5 years as be making a grand upgrade to v1 in 6 months. I also think that the only user of the API for a long time will be me (or more precisely my site https://lingopolo.org/), so even if we tweek the returned results in a non backward-compatible way, I'm OK with that. Still, it seems a reasonable request, so I have made the required changes.